MicroProfile-Health/tck-runner/pom.xml

```diff
@@ -69,18 +69,29 @@
             <version>${microprofile.health.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>tck-arquillian-extension</artifactId>
-            <version>${project.version}</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>fish.payara.arquillian</groupId>
             <artifactId>payara-client-ee7</artifactId>
             <version>${payara.arquillian.container.version}</version>
             <scope>test</scope>
         </dependency>
+        
+        <!-- CAUSES ISSUES WITH MICRO MANAGED -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>tck-arquillian-extension</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+            <exclusions>
```

Comment:
Why does the artifact include this in the first place? That seems like the root of the problem

Reply:
You're not wrong, and honestly I can't tell you - dependency isn't stated anywhere in the parent pom and NetBeans won't remove it so this was the best way I could think of to resolve the issue

Reply:
As a hint, it'll be an inherited dependency. I would suggest looking further up the tree and potentially moving the dependency further down if it's possible

Reply:
That's what I mean, the top level pom (TCK-parent) doesn't have a generally available dependency on that container, from having looked at it, it appears to be coming in with an external dependency group that's also importing required test dependencies

Reply:
I'll have another look and confirm